File: Python Word Examples/ConvertApiDemo/convertToVectorgraph.py
import spirecloudword
from spirecloudword import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudword.api.convert_api.ConvertApi(configuration)

file_name = "ConvertToVectorgraph.docx"
image_type = "wmf" #Available values : wmf, svg
dest_file_path = 'output/ConvertToVectorgraph_WMF/'
extra = {'folder': 'input'}
api.convert_to_vectorgraph(file_name, image_type, dest_file_path, **extra)

File: JavaScript Word Examples/ConvertApiDemo/convertToVectorgraph.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.ConvertApi(apiClient);

    var name = "Convert.docx";
    var imageType = "wmf"; //Available values : wmf, svg
    var destFolder = 'output/ConvertToVectorgraph_WMF/';
    var opt = { 'folder': 'input' };
    instance.convertToVectorgraph(name, imageType, destFolder, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();